data/curated/wa-cannabis/meta.yaml

```
@@ -0,0 +1,14 @@
+title: "Washington Cannabis Industry (2024–2025)"
+source: 
+  title: "Washington State Liquor and Cannabis Board (WSLCB) Reporting Data"
+  url: "https://lcb.wa.gov/"
```

The yaml is only ever used by an automated process. You need to follow the structure exactly. Please add an example of the data being used. If none exists, create a simple repo in your personal github that demonstrates usage.
```suggestion
article:
  title: TITLE OF ARTICLE THAT USES OR DESCRIBES THE DATASET
  url: URL OF THAT ARTICLE
data_source: 
  title: "Washington State Liquor and Cannabis Board (WSLCB) Reporting Data"
  url: "https://lcb.wa.gov/"
```